images/win/scripts/Installers/Validate-Julia.ps1

```diff
@@ -0,0 +1,25 @@
+################################################################################
+##  File:  Validate-Julia.ps1
+##  Desc:  Validate Julia
+################################################################################
+
+# Verify that julia.exe is on the path
+if(Get-Command -Name 'julia')
+{
+    Write-Host "$(julia --version) is on the path."
+}
+else
+{
+    Write-Host "Julia is not on the path."
+    exit 1
+}
+
+# Add description of the software to Markdown
+$SoftwareName = "Julia (x64)"
+$juliaVersion = $(julia --version) -match "\d+\.\d+\.\d+"
```

Comment:
```
PS > $(julia --version) -match "\d+\.\d+\.\d+"
True
```
I think it should be:
`$(julia --version).split() -match "\d+.\d+\.\d+"`

Reply:
Fixed in https://github.com/actions/virtual-environments/pull/386/commits/68fb8ac37ca6a4778ce938bf5d87fd1fbe7b084e